selfdrive/car/interfaces.py

```diff
@@ -101,6 +103,23 @@ def get_steer_feedforward_default(desired_angle, v_ego):
   def get_steer_feedforward_function(self):
     return self.get_steer_feedforward_default
 
+  @staticmethod
+  def torque_from_lateral_accel_linear(lateral_accel_value, torque_params, lateral_accel_error=None, lateral_accel_deadzone=None, friction_compensation=False):
+    # The default is a linear relationship between torque and lateral acceleration (accounting for road roll and steering friction)
+    if friction_compensation:
+      assert (lateral_accel_error is not None) and (lateral_accel_deadzone is not None)
+      friction = interp(
+        apply_deadzone(lateral_accel_error, lateral_accel_deadzone),
+        [-FRICTION_THRESHOLD, FRICTION_THRESHOLD],
+        [-torque_params['friction'], torque_params['friction']]
+      )
+    else:
+      friction = 0.0
+    return (lateral_accel_value / torque_params['latAccelFactor']) + friction
+
+  def torque_from_lateral_accel(self):
+    return self.torque_from_lateral_accel_linear
```

Comment:
add type hints for the expected callback, like this https://github.com/commaai/openpilot/blob/8ae3199578b3f771396cd35315ce51b7cce27e49/selfdrive/controls/lib/events.py#L208